CHANGELOG.md

```diff
@@ -20,10 +20,74 @@ This section includes a summary of user-facing changes since the last release. F
 changes since the last release, see the [diff on GitHub][unreleased].
 
 <!-- Unreleased comparison link -->
-[unreleased]: https://github.com/PowerShell/DSC/compare/v3.0.0-alpha.2...main
+[unreleased]: https://github.com/PowerShell/DSC/compare/v3.0.0-alpha.3...main
 
 <!-- Add entries between releases under the appropriate section heading here  -->
 
+## [v3.0.0-alpha.3][release-v3.0.0-alpha.3] - 2023-09-26
+
+This section includes a summary of changes for the `alpha.3` release. For the full list of changes
+in this release, see the [diff on GitHub][compare-v3.0.0-alpha.3].
+
+<!-- Release links -->
+[release-v3.0.0-alpha.3]: https://github.com/PowerShell/DSC/releases/tag/v3.0.0-alpha.3 "Link to the DSC v3.0.0-alpha.3 release on GitHub"
+[compare-v3.0.0-alpha.3]: https://github.com/PowerShell/DSC/compare/v3.0.0-alpha.2...v3.0.0-alpha.3
+
+### Changed
+
+- Replaced the `manifestVersion` property for resource manifests with [`$schema`][15]. Instead of
+  specifying a semantic version, resources need to indicate which canonical schema DSC should use
+  to validate and process the manifest.
+
+  <details><summary>Related work items</summary>
+
+  - Issues: [#127][#127]
+  - PRs: [#199][#199]
+
+  </details>
+
+- Updated the `preTest` property for the `set` command in resource manifests to
+  [`implementsPretest`][16] to more make the manifest easier to read and understand.
+
+  <details><summary>Related work items</summary>
+
+  - PRs: [#197][#197]
+
+  </details>
+
+- The [dsc resource set][17] command no longer tests the resource
+  instance before invoking the `set` operation. This simplifies the behavior for the command and
+  adheres more accurately to the implied contract for directly invoking a resource with DSC.
+
+  <details><summary>Related work items</summary>
+
+  - Issues: [#98][#98]
+  - PRs: [#197][#197]
+
+  </details>
+
+- Replaced the `args` option with `env` for defining how a command-based resource expects to
+  receive input for the [get][18], [set][19], and [test][20] commands in the resource manifest.
+
+  The `args` option wasn't implemented. Instead, resource authors can set the `input` property to
+  `env` to indicate that the resource expects input as environmental variables.
+
+  <details><summary>Related work items</summary>
+
+  - PRs: [#198][#198]
+
+  </details>
+
+- The `input` property for the [get][18], [set][19], and [test][20] commands in a resource manifest
+  no longer has a default value. Instead, when a resource doesn't define `input` for a command, DSC
+  doesn't send any input to the resource for that command.
```

Comment:
@SteveL-MSFT - this was my understanding from the PR, but I'm not _entirely_ sure - does this change actually only affect `get`? If so, I can update this PR and #203 to reflect that.